examples/ordered-options.mjs

```diff
@@ -0,0 +1,35 @@
+// Now might I enforce that two flags are specified in a specific order?
+
+import { parseArgs } from '../index.js';
```

Comment:
Other examples are `cjs`; is the change deliberate?

Reply:
Yes, partly as a proof of concept. I started out thinking I would do each example twice but that seemed overkill, so thinking might mix the examples with `cjs`/`mjs` (and only changed over one so far).

Reply:
It's only one line which is different, so I don't think it's worth putting much effort in. One seems like plenty.